### ISSUE TITLE
Cannot install pystan with new anaconda version

### ISSUE THREAD

**Issue by zstrinka** [stan-dev/pystan#299]
#### Summary:
Anaconda updated to Python 3.6 last week. While I was able to get pystan working fine on my windows server when I installed python 3.5.3, I was unable to get it to install with the Anaconda version of python.

#### Description:
On both a Windows 2016 server and a Windows 2008 server I got an error message when attempting to install pystan with the latest Anaconda distribution for Python 3.6. The specific error message referred to an issue with setuptools, despite being up to date. When I instead installed Python 3.5.3 along with numpy and Cython, both computers were able to install pystan without issue.

#### Reproducible Steps:

I believe this is an issue for windows and the latest Anaconda distribution

#### Current Output:
An error message that pystan cannot be installed, despite having all the appropriate prerequisites.

#### Expected Output:
That pystan would install.

#### PyStan Version:
2.14.0.0 installed using pip.

#### Python Version:
3.6 has issues, 3.5.3 does not
#### Operating System:
Windows 2008 server version and Windows 2016 server

**Comment by ahartikainen** [comment]
I have had also some problems lately on windows 7 with Visual Studio 2015 (v14), python 3.5/3.6 installed with latest miniconda.

Pip install does not work, but pip install git+https://github.com/stan-dev/pystan works. So you could first try that.

After pip install is succesfull, 8-schools compiles and samples fine, but one with GP code fails. Error was "Internal compiler error  ... something" and there was some problems with 'this' method / command on some Eigen or Boost file. There was somewhere said that vc14 forces c++11, but I don't really know if that is relevant.

Can give better description when on that windows machine after weekend. I am still debugging so have not created any issues yet.

**Comment by ariddell** [comment]
Thanks for the report @zstrinka . The supported setups under Windows are limited right now: http://pystan.readthedocs.io/en/latest/windows.html 

Can you verify you've got the right compiler installed?

**Comment by zstrinka** [comment]
@aridell I did have the right compiler installed on the computer. I am still pretty new to Python, so I had a hard time figuring out if there was something I could have modified beyond just installing visual studio 2015. I am pretty sure it had to do with Anaconda though. When I just installed Python 3.5 (and Cython and Numpy) instead of all of Anaconda, I could immediately install pystan without any more fiddling with visual studio.
